.github/workflows/release.yaml

```diff
@@ -0,0 +1,30 @@
+name: Release
+
+on: workflow_dispatch
+
+jobs:
+    release:
+        name: Release
+        runs-on: ubuntu-latest
+        steps:
+            - name: Checkout
+              uses: actions/checkout@v3
+            - name: Get the version
+              id: package-version
+              uses: martinbeentjes/npm-get-version-action@v1.3.1
+            - name: Setup Node
+              uses: actions/setup-node@v3
+              with:
+                  node-version-file: .nvmrc
+                  cache: "npm"
+                  registry-url: "https://registry.npmjs.org"
+            - name: Install
+              run: npm install
+            - name: Test
+              run: npx nx run-many -t test
+            - name: Build & Publish
+              run:
+                  npx nx run-many -t publish --registry=npm --ver=${{ format('{0}',
+                  steps.package-version.outputs.current-version)}} --tag=latest
```

Comment:
You might want to match current release workflow from corelibrary, but there is only one package in this repo so there shouldn't be any issues with current version.